modules/ROOT/pages/visualizer-app-network.adoc

```diff
@@ -52,22 +51,26 @@ https://en.wikipedia.org/wiki/Metric_prefix[Here is a quick review of metric pre
 
 == View Information About a Node in Your Network
 
-When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are grayed out. Anypoint Visualizer displays information about the node in the panel on the right. This panel is divided into the following sections:
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are grayed out. Anypoint Visualizer displays information about the node in a *Details/Monitoring* card. This card is divided into the following tabs:
 
+The *Details* tab contains:
 * *Details*: Provides information about the node.
+* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can edit the node's display name or change the layer and tags from this section.
+
+The *Monitoring* tab contains:
 * *Metrics summary*: Provides monitoring metrics for the selected node including Errors count, avg. response time, avg. throughput, avg. CPU consumption, and avg. memory utilization.
 * *Metrics chart*: Shows the time-series chart of the selected metric in the *Metric* drop-down.
-* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can edit the node's display name or change the layer and tags from this section.
 
-You can search for a node (service) in the *Search services* text field. Services containing your search in any part of their name are highlighted, while others fade.
+
+You can search for a node (service) in the *Filter services* text field. Services containing your search in any part of their name are highlighted, while others fade.
```

Comment:
Mockup is wrong, sorry. It's still Search services

Reply:
Accepted.